projects/media-viewer/src/lib/toolbar/main-toolbar/search-bar/search-bar.component.ts

```diff
@@ -40,11 +40,14 @@ export class SearchBarComponent implements OnInit, OnDestroy {
 
   @HostListener('window:keydown', ['$event'])
```

Comment:
should this code be moved from here then? into the key-listener-service

Reply:
HostListeners must be a part of a component as far as I can tell.

Reply:
ah ok, but it seems like we're doing key binding and matching here, so should this delegate to the service like the media-viewer does?

Reply:
Sry - this HostListener was not added by me. I believe this particular Listener could be removed after we add the global handling of key strokes.